marimo/__init__.py

```diff
@@ -84,9 +84,8 @@
     "video",
     "vstack",
     "watch",
+    "__version__",
 ]
-__version__ = "0.14.13"
```

Comment:
I suppose dynamic versioning cleans things up, but wouldn't mind some marker in source for reference

Reply:
`pyproject.toml` is the canonical, static source of truth and required in both `sdist` and `wheel`. uv build-system doesn't allow for 'dynamic' metadata (like deriving a version from python source) because they think it's an anti-pattern to have dynamic package metadata. Importing from `importlib.metadata` is reading static metadata.